projects/storefrontlib/src/cms-components/myaccount/order/amend-order/amend-order-actions/amend-order-actions.component.ts

```diff
@@ -12,9 +14,23 @@ import {
 })
 export class AmendOrderActionsComponent {
   @Input() orderCode: string;
-  @Input() isValid: string;
+  @Input() form: FormGroup;
```

Comment:
According to the latest form changes, please name this form correctly. Read more here: https://github.com/SAP/spartacus/blob/develop/docs/migration/2_0.md#naming-convention